tests/e2e/tests/generate/class.ts

```diff
@@ -0,0 +1,17 @@
+import {join} from 'path';
+import {ng} from '../../utils/process';
+import {expectFileToExist} from '../../utils/fs';
+
+
+export default function() {
+  // Create the pipe in the same directory.
+  const classDir = join('src', 'app');
+
+  return ng('generate', 'class', 'test-class')
+    .then(() => expectFileToExist(classDir))
+    .then(() => expectFileToExist(join(classDir, 'test-class.ts')))
+    .then(() => expectFileToExist(join(classDir, 'test-class.spec.ts')))
```

Comment:
This just changed to default to not having a spec in https://github.com/angular/angular-cli/pull/2475 I believe.